src/apps/material/helper.ts

```diff
@@ -11,7 +11,13 @@ import {
 import { UseTextFunction } from "../../core/utils/text";
 import { Manifestation, Work } from "../../core/utils/types/entities";
 
-export const getWorkManifestation = (work: Work) => {
+export const getWorkManifestation = (
+  work: Work,
+  type: "bestRepresentation" | "latest"
+) => {
+  if (type === "bestRepresentation") {
```

Comment:
I think we could create an object const for bestRepresentation" and "latest" globally.
And then it should be possible to make a type of it as well.
Maybe the iidiom could be called ManifestationType?